test/unit/entity/acl.test.js

```diff
@@ -0,0 +1,319 @@
+import { Acl } from 'src/entity';
+import { randomString } from 'src/utils';
+import expect from 'expect';
+
+describe('Acl', function() {
```

Comment:
I think this test would be a bit redundant. Since you can mock whatever you want that mock wouldn't be anymore correct then what I have. I do have unit tests on the user class that check `emailVerification` and other properties on the `acl` and `kmd` properties.